internals/core_models.py

```diff
@@ -954,3 +954,206 @@ def put(self, notify=True, **kwargs):
   anticipated_spec_changes = ndb.StringProperty()
 
   finch_url = ndb.StringProperty()
+
+
+# Note: This class is not used yet.
+class FeatureEntry(ndb.Model):  # Copy from Feature
+  """This is the main representation of a feature that we are tracking."""
+
+  # Metadata: Creation and updates.
+  created = ndb.DateTimeProperty(auto_now_add=True)
+  updated = ndb.DateTimeProperty()
+  accurate_as_of = ndb.DateTimeProperty()
+  creator = ndb.StringProperty()
+  updater = ndb.StringProperty()
+
+  # Metadata: Access controls
+  owners = ndb.StringProperty(repeated=True)  # copy from owner
+  editors = ndb.StringProperty(repeated=True)
+  unlisted = ndb.BooleanProperty(default=False)
+  deleted = ndb.BooleanProperty(default=False)
+
+  # Descriptive info.
+  name = ndb.StringProperty(required=True)
+  summary = ndb.TextProperty(required=True)
+  category = ndb.IntegerProperty(required=True)
+  blink_components = ndb.StringProperty(repeated=True)
+  star_count = ndb.IntegerProperty(default=0)
+  search_tags = ndb.StringProperty(repeated=True)
+  feature_notes = ndb.TextProperty()  # copy from comments
+
+  # Metadata: Process information
+  feature_type = ndb.IntegerProperty(default=FEATURE_TYPE_INCUBATE_ID)
+  intent_stage = ndb.IntegerProperty(default=0)
+  bug_url = ndb.StringProperty()
+  launch_bug_url = ndb.StringProperty()
+
+  # Implementation in Chrome
+  impl_status_chrome = ndb.IntegerProperty(required=True)
+  flag_name = ndb.StringProperty()
```

Comment:
This should be developer flag name or something similar. I have to frequently tell people feature owners that this does NOT apply to finch flags.

Reply:
I need to better understand the distinction.  I'll ask this question on webstatus@.

Reply:
Finch flags are controlled by Chrome exclusively. Having a Finch flag does NOT constitute a developer trial. developer trial flags are set on chrome://flags or via a command line. Beyond programming a default value, Chrome has NO control over how they're set. With Finch we can turn features on or off for all Chrome via in-house infrastructure. To change a dev trial default, we have to release a new version of Chrome.